replay/models/ddpg.py

```diff
@@ -444,12 +537,18 @@ def __init__(
         self,
         noise_sigma: float = 0.2,
         noise_theta: float = 0.05,
-        noise_type: str = "ou",
+        noise_type: str = "gauss",
         seed: int = 9,
         user_num: int = 10,
         item_num: int = 10,
         log_dir: str = "logs/tmp",
         exact_embeddings_size=True,
+        n_critics_head: int = 10,
```

Comment:
could you please update descriptions? e.g. add link to BayesUCBDQN paper, indicate which arguments are related to it